packages/modules/order/src/joiner-config.ts

```diff
@@ -37,6 +37,7 @@ export const joinerConfig: ModuleJoinerConfig = {
       name: ["return_reason", "return_reasons"],
       args: {
         entity: ReturnReason.name,
+        methodSuffix: pluralize(ReturnReason.name),
```

Comment:
@carlos-r-l-rodrigues do you think it would be wize by default to do that? if the entity if provided and it is not the first ailas then we do that under the hood? convention being the main entity is always the first

Reply:
Yeah, it is alright to do that.
Maybe we could even infer the methodSuffix in the remote query from the property entity. and if it is different we can specify it.
But not in this PR.